src/main/java/org/assertj/core/api/ObjectEnumerableAssert.java

```diff
@@ -57,6 +57,8 @@
   /**
    * Verifies that the actual group contains only the given values and nothing else, in any order.
    * <p>
+   * Use {@link #isSubsetOf(Object[])} to check if actual is a subset of given iterable
```

Comment:
`to check if` -> `to check that`, the link point to `isSubsetOf` that takes a vararg and not an iterable

Reply:
add an example in the `//assertion will fail` section to show that this assertion fails because of the missing element `"d"`.
```
assertThat(abc).containsOnly("a", "b", "c", "d");
```